### DISCUSSION TITLE
Scan encountered an exception

### DISCUSSION THREAD

**Discussion by configworld768** [projectdiscovery/nuclei#930] [Q&A] [answered]
**Describe the bug**
I did not specify the ssh-related template, but the ssh template was used when scanning
![image](https://user-images.githubusercontent.com/39177268/129165281-bb6c3083-fd94-4110-8d8c-213b0b261b2b.png)

**Nuclei version**

```
                    __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   2.4.3

                projectdiscovery.io

[INF] Current Version: 2.4.3
```

**Screenshot of the error or bug**
![image](https://user-images.githubusercontent.com/39177268/129165590-7331805f-3c8f-4128-a1aa-ca25922b9224.png)

**Comment by ehsandeep** [comment, marked as answer]
@bigzeroo you need to use `-t` flag to specify templates, in this case, you missed `-t` flag, as a result, the default behavior of nuclei is to run all templates.

Valid example to run specific template:-

```
nuclei -u 10.50.20.52:6401 -t network/exposed-redis.yaml
```
